packages/api/src/task/scheduler.ts

```diff
@@ -539,10 +539,44 @@ export class TaskScheduler {
       if (typeof asset === "string") {
         asset = await db.asset.get(asset);
       }
+
+      let phase = asset.status?.phase;
+      // Prevent bump of updatedAt if phase isn't getting updated
+      let updatedAt = asset.status?.updatedAt;
+
+      if (phase == "ready") {
+        // If the asset is ready, we need to schedule deletion
```

Comment:
I wonder if we want to trigger this for failed assets too. Maybe there are left overs?